projects/_posts/2020-03-01-teleirc.md

```diff
@@ -32,7 +32,7 @@ Today, it is used by communities around the world.
 ## Resources and links
 
 * Community:
-    * [IRC channel](https://webchat.freenode.net/?channels=rit-lug-teleirc)
+    * [IRC channel](https://web.libera.chat/#rit-lug-teleirc)
```

Comment:
So, TeleIRC is actually still on freenode for the time being. Everything looks great except this portion here.

Reply:
oh ok. ill revert this section